packages/react/src/patterns/blocks/ContentBlockSegmented/README.md

```diff
@@ -0,0 +1,76 @@
+# Content Block - Segmented
+
+> The Content Block - Segmented pattern allows for larger groups of content to
+> be presented at once.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import fonts once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ContentBlockSegmented } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <ContentBlockSegmented
+      copy={copy}
+      cta={cta}
+      heading={heading}
+      image={image}
+      items={items}
```

Comment:
Do we have anything to indicate the shape of this items? Can we point them to a data object or something?